Add Storybook

lauthieb opened the pull request:
<!-- Congrats on creating an Awesome list! 🎉 -->

<!-- Please fill in the below placeholders -->

**https://github.com/lauthieb/awesome-storybook**

**awesome-storybook** is a collection of awesome @storybooks resources like tutorials, presentations, examples and blog posts.
Many people in the community already use it and I really hope it can be added to this repository.

**Awesome-Lint Passing**

> awesome-lint
✔ Linting

**PR reviews**
- #1563 
- #1566
- #1559

**What is Storybook?**
[Storybook](https://storybook.js.org) is a user interface development environment and playground for UI components. The tool enables developers to create components independently and showcase components interactively in an isolated development environment.
Storybook runs outside of the main app so users can develop UI components in isolation without worrying about app specific dependencies and requirements._

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The suggested Awesome list complies with the below requirements.


## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The heading title of your list should be in the following format: `# Awesome Name of List`
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# `
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- **If you use a license badge, it should be SVG, not PNG.**
	- To verify that you've read all the guidelines, please add comment with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

sindresorhus commented:
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

lauthieb commented:
unicorn

lauthieb commented:
Hi @sindresorhus, thanks for your answer!
For me it's good now.
- I've fixed all linting errors with awesome-lint + add it as a `test` script in package.json and activate Travis CI to lint on new commits and pull requests.
- Review 3 PR's (#1563, #1566, #1559)
- Add the word `unicorn` in comment
- Add some descriptions on my awesome items
- Add #readme in the URL

I hope you will approve it 🙏:smiley:

sindresorhus commented:
> The heading title of your list should be in the following format: # Awesome Name of List

Not done.

---

  README.md:30:47
  ✖  30:47  Text "Github" should be written as "GitHub"  remark-lint:awesome/spell-check

  1 error

---

It's weird to have so much stuff under the `Resources` section. I would make all those sections there top-level instead. For example, why is `Blog posts` under `Resources`, but not `Examples`. They are both resources.

sindresorhus commented:
I think the items in the `Blog posts` section could use a description with a short summary of the post.

sindresorhus commented:
For the descriptions of the items in https://github.com/lauthieb/awesome-storybook#official-resources, it's moot to mention `Storybook` as it's implied. Most of those descriptions are moot anyway as they only repeat the title without giving any additional information.

---

> GitHub repo - Storybook's organization on Github.

This one is not even correct. The title is to a GitHub repo, the link and description is about a GitHub organization.

lauthieb commented:
Hello @sindresorhus, thanks a lot for your reactivity and sorry for those few omissions...

I've fixed all your remarks 👍 

✅ The heading title of your list should be in the following format: # Awesome Name of List
✅ Linting with 0 error
✅ So much stuff under the `Resources` section. I made all those sections in a top-level instead.
✅ On the items of the `Blog posts` section, I've added a description with a short summary of each post.
✅ Most of those descriptions in `Official resources` are moot anyway as they only repeat the title without giving any additional information. I've removed them.
✅ Wording on "Github repo" to "Github organization"

Sorry again. I really hope my PR will suit you now and will be able to get in 🙏

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1130769686252015616

MIV1818196 commented:
af691abc79ca1a110371ea81d61a4ed4c838f09c
